docs/future-work/motor-system-improvements/policy-to-quickly-move-to-a-new-object.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Policy to quickly move to a new object
+---
+
+When exploring an environment with multiple objects (including components of a compositional object), it is beneficial to quickly move to a new object when the current one has been recognized, so as to rapidly build up a model of the outside world. 
+
+It would be useful to have a policy that uses a mixture of model-free components (e.g. saliency map) and model-based components (learned relations of objects to one another) to make a decision about where to move next in such an instance.
```

Comment:
```suggestion
It would be useful to have a policy that uses a mixture of model-free components (e.g. saliency map) and model-based components (learned relations of sub-objects to one another in a higher-level LM) to make a decision about where to move next in such an instance.
```